### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by pravins**
TODO: Steps to install and license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
+* [@sshedmake](https://github.com/sshedmake)
+* [@sourabh-upadhyay](https://github.com/sourabh-upadhyay)
+* [@tpardeshi](https://github.com/tpardeshi)
+
+## Contributing
+
+Feel free to dive in! [Open an issue](https://github.com/lingostack/lingoqa/issues/new) or submit PRs.
+
+## License
+GNU General Public License, version 2 (GPLv2)
```
Comment (rahulbajaj0509) [inline README.md:78]: Can we change this to `version 3 (GPLv3)`, in reference to the license PR (https://github.com/lingostack/lingoqa/pull/29)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,78 @@
...
+5. $rake db:migrate
+6. $rails s
+
+## Usage
+
+1. Generate screenshots of Product UI in multiple languages. 
+2. Upload screenshots in Amazon S3 Bucket
+3. Configure LingoQA for your project.
+
+## Maintainers
```
Comment (rahulbajaj0509) [inline README.md:63]: It would be nice to name this as `Contributors`

Comment (pravins) [reply README.md:63]: Agree

File: README.md (hunk 1 above)
Comment (pravins) [reply README.md:78]: I think, lets do it GPLv3+, so automatically enhancements in future will become part of this project license.

Comment (rahulbajaj0509) [reply README.md:78]: The file added has GPL v2 version.

File: README.md (hunk 2 above)
Comment (pravins) [reply README.md]: Done

**Comment by pravins** [comment]
All changes has been done, feel free to merge now :)

**Comment by rahulbajaj0509** [comment]
Thanks @pravins! merging now :)

**Pull request merged**